modules/tinymce/src/core/test/ts/browser/keyboard/PageUpDownKeysTest.ts

```diff
@@ -76,7 +77,7 @@ describe('browser.tinymce.core.keyboard.PageUpDownKeyTest', () => {
       }
     });
 
-    it('caret is not at the beginning/end of an inline element, not escape', () => {
+    it('Caret wont move if it is not at the beginning/end of the inline element', () => {
```

Comment:
This is in the middle? It should move out if it ends up at the end of the link.

Reply:
yes, it's in the middle, so it shouldn't move